includes/class.llms.certificates.php

```diff
@@ -175,6 +177,7 @@ public function get_export( $certificate_id, $use_cache = false ) {
 	 * @since 3.18.0
 	 * @since 3.24.3 Unknown.
 	 * @since 3.37.3 Refactored method into multiple functions.
+	 * @since [version] If `$this->scrape_certificate()` generates a `WP_Error` early return it.
 	 *
 	 * @param int $certificate_id WP_Post ID of the earned certificate.
 	 * @return string
```

Comment:
The return hasn't technically changed (not that it matters with the private method) but we should document it properly at least!

```suggestion
	 * @return WP_Error|string HTML of the certificate on success, otherwise an error object.
```

Reply:
Yeah you're right, I missed it because as you said this PR didn't change the return :/